webui.py

```diff
@@ -17,7 +17,10 @@
     cmd_flags_path = os.path.join(script_dir, "CMD_FLAGS.txt")
     if os.path.exists(cmd_flags_path):
         with open(cmd_flags_path, 'r') as f:
-            CMD_FLAGS = ' '.join(line.strip() for line in f.read().splitlines() if line.strip())
+            CMD_FLAGS = ' '.join(line.strip() for line in f.read().splitlines() if line.strip() and not line.strip().startswith('#'))
```

Comment:
```suggestion
            CMD_FLAGS = ' '.join(line.strip() for line in f if line.strip() and not line.strip().startswith('#'))
```
You don't need `read()` to iterate lines in a file.

Reply:
Thanks @missionfloyd - that's better now